CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.7.0 - 2020-09-10
```

Comment:
Please remove date as we don't know yet when will it be released. Just put a `TODO` or `To be released` next to the version.

Reply:
done